.github/workflows/on-pull-request.yml

```diff
@@ -119,6 +119,9 @@ jobs:
           sudo mv lychee /usr/local/bin/
           lychee --version
 
+      - name: Wait for preview deployment to stabilize
+        run: sleep 180
```

Comment:
This is the main fix for the PR. The preview env needs time to be ready before we check for broken links